test/test_genbank_sbol3_direct.py

```diff
@@ -15,96 +15,133 @@ class TestGenBankSBOL3(unittest.TestCase):
     # Create converter instance
     converter = GenBank_SBOL3_Converter()
 
-    def test_gbtosbol3_1(self):
-        """Test conversion of a simple genbank file with a single sequence"""
-        SAMPLE_GENBANK_FILE_1 = Path(__file__).parent / "test_files" / "BBa_J23101.gb"
-        SAMPLE_SBOL3_FILE_1 = (
-            Path(__file__).parent
-            / "test_files"
-            / "BBa_J23101_from_genbank_to_sbol3_direct.nt"
-        )
-        sbol3.set_namespace(TEST_NAMESPACE)
-        TEST_OUTPUT_SBOL3 = str(SAMPLE_SBOL3_FILE_1) + ".test"
+    def _test_genbank_to_sbol3(self, SAMPLE_SBOL3_FILE: Path, SAMPLE_GENBANK_FILE: Path):
```

Comment:
I'm glad to see a helper function here. This is a good software design choice.

Reply:
Yep, thanks